upgrading/extending-daml-models/index.json

```diff
@@ -0,0 +1,43 @@
+{
+  "title": "Extending DAML models",
+  "description": "Extend a DAML model with a feature contained in a new package and upload it to a running ledger.",
+  "difficulty": "Intermediate",
+  "time": "20 minutes",
+  "details": {
+    "steps": [
+      {
+        "title": "Package IDs",
+        "text": "1_package_ids.md",
+        "code": "init.sh"
```

Comment:
You can move the init.sh steps to the intro slide so that user doesn't see the commands (e.g., I've done it [here](https://github.com/digital-asset/daml-katacoda/blob/master/getting-started/your-first-feature/index.json))

Reply:
I kind of like showing it to the user just as repetition and that you see that there is no magic going on.

Reply:
Depends how you look at it. If I would be working locally I would do creare-daml-app only once. If nothing else the `rm -rf create-daml-app` is just for Katacoda reasons. 

Leave it up to you